RiotSharp/Requester.cs

```diff
@@ -1,90 +1,124 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.IO;
+using System.Linq;
 using System.Net;
+using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace RiotSharp
 {
     class Requester
     {
-        protected string rootDomain;
+        protected string RootDomain;
+        private readonly HttpClient _httpClient;
```

Comment:
please use only camel case without the starting `_` to be coherent with the rest of the project, thanks